src/main/java/graphql/validation/rules/DeferDirectiveLabel.java

```diff
@@ -0,0 +1,41 @@
+package graphql.validation.rules;
+
+import graphql.Internal;
+import graphql.language.Argument;
+import graphql.language.Directive;
+import graphql.language.Node;
+import graphql.language.StringValue;
+import graphql.validation.AbstractRule;
+import graphql.validation.ValidationContext;
+import graphql.validation.ValidationErrorCollector;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+import static graphql.validation.ValidationErrorType.DuplicateArgumentNames;
+
+@Internal
```

Comment:
can we put in links to the proposed spec PR.  Later we can update it to the real thing when it becomes real